(feat) option Fix to context menu

alyxme opened the pull request:
Add option to Fix vehicle via context menu
Closes #1954

alyxme committed:
Update context_menu_service.hpp

alyxme committed:
Add FIX vehicle

Fix option to fix vehicle

Tayipc38 commented on src/services/context_menu/context_menu_service.hpp:
```diff
@@ -72,6 +72,18 @@ namespace big
+				            VEHICLE::SET_VEHICLE_DIRT_LEVEL(m_handle, 0.f);
+			            }
+			            else
+				            g_notification_service->push_warning("Toxic", "Failed to take control of vehicle.");
```
I don't think this falls under toxic does it?

alyxme replied:
Could it be just "Warning" instead of "Toxic" notice?
Also Id rearrange list to remove "explode" from atop to not accidentaly blowup something like it happen so often.

alyxme replied:
Yes it saves, but for current game session only. So I did rearrange. Explode entity is strange behavior tbh. I moved it to bottom of list. Also I replace identity with outfit because it is more useful than frustrating if click on it and can't be revert until rejoin.

alyxme committed:
Update to context menu

Push Warning if FIX VEHICLE cant take controll
Replace Steal identity with Steal outfit
Move Explode enitity to last in list

alyxme committed:
Merge branch 'master' into master

alyxme committed:
Rearrange context list

Rearranged context list moving Copy hash at top to ease operation with entities.
